src/main/java/uk/gov/digital/ho/proving/income/audit/statistics/AuditClientEndpointProperties.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.digital.ho.proving.income.audit.statistics;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+
+@Configuration
+@ConfigurationProperties(prefix = "pttg.audit")
+@NoArgsConstructor
+@Setter
```

Comment:
Do you have to have a no-args constructor and setters?

Reply:
Yes. That's how Spring instantiates the class. It's also the reason why we can't use fluent getters.